src/main/webapp/script.js

```diff
@@ -148,17 +163,24 @@ function addHiddenPoiFormTrigger() {
     const poiInputs = document.getElementsByClassName('poi-input');
     let count = 1;
     Array.prototype.forEach.call(poiInputs, (poiInput) => {
-      $('<input>').attr('type', 'hidden')
-      .attr('name', 'poi-' + count)
-      .attr('value', poiInput.name)
-      .appendTo('#startTripForm');
+      let poiNumber = 'poi-' + count;
+      addHiddenInput(poiNumber, poiNumber, poiInput.name);
```

Comment:
not sure I understand why this should be a helper method, are you planning to use it elsewhere? Seems very specific to this situation, and I don't know if it increases the clarity. I was actually confused what you meant by 'hidden input' until I read the code itself.

Not a huge deal, it could be a personal preference. So if you feel strongly about keeping it this way, that's fine with me

Reply:
That makes sense -- done!